vpr/src/place/place.cpp

```diff
@@ -1252,9 +1251,11 @@ static float starting_t(const t_annealing_state* state, t_placer_costs* costs, t
     VTR_LOG("std_dev: %g, average cost: %g, starting temp: %g\n", std_dev, av, 20. * std_dev);
 #endif
 
-    /* Set the initial temperature to 20 times the standard of deviation */
+    /* Set the initial temperature to the standard of deviation divided by 64 */
```

Comment:
and so that the initial placement is not completely destroyed (lets us have a smarter initial placement and fine-tune it with the anneal).